workshops/personal_website/README.md

```diff
@@ -163,331 +122,204 @@ Every HTML file follows this template:
+heading, and my code looks like this:
 
 ```html
 <!DOCTYPE html>
 <html>
     <head>
-            ← this section is called the "head" of the HTML document
     </head>
     <body>
-            ← this section is called the "body" of the HTML document
+        <img src="https://github.com/hackclub/dinosaurs/raw/master/smart_dinosaur_docs.png"/>
+        <h1>Prophet Orpheus</h1>
+        <p>Coder Dino</p>
+        <p>Will code for food</p>
     </body>
 </html>
 ```
 
-- Everything between the `<head>` and `</head>` tags is called the **head** of
-  the page. `<head>` is considered an "opening tag" because it "opens" the head
-  section. Since `</head>` starts with `</` (instead of just `<`), it is
-  considered a "closing tag" and "closes", or indicates the end of, the head
-  section.
-- Similarly, everything between the `<body>` and `</body>` tags is called the
-  **body** of the page. `<body>` is the opening tag and `</body>` is the closing
-  tag. Together, the opening tag and the closing tag compromise the `body`
-  element.
-- Note: most HTML elements have an _opening tag_ and a _closing tag_. You'll
-  want to get used to this pattern :smile:.
-
-### 4) Opening the Website Preview
-
-Let's see what our website looks like so far:
-
-1. First, save the file by clicking **"File"** _(on the top right)_ → **"Save"**
-   (or use the shortcut **CTRL + S / Command + S**)
-2. Preview what the website looks like by clicking **"Preview"** → **"Live
-   Preview File"**
+Now, your image may either be too big, or too small. Fret not. CSS will allow
+you to manipulate the styling of your page in all your needs.
 
-> ![](img/c9_live_preview.gif)
+## Part III: The CSS File
 
-As you can see, the page is still blank. This is because we haven't added any
-**content** in the **`body`** section yet. Let's add some!
+So what is CSS? CSS is a language used for styling elements, hence its name,
+Cascading Style Sheets.
 
-### 5) Adding a Name to the Body of the HTML
+While HTML oversees the content and structure of the content, CSS is how you'll
+specify how you'd like your content to look, and with it, you can set things
+like colors, spacing, and more.
 
-Write your name in the **body** of the HTML page (between **`<body>`** and
-**`</body>`**).
-
-> ![](img/html_add_name.gif)
-
-Notice that that the HTML we write is immediately updated in the preview. That's
-why it's called a **"Live Preview"**
-
-### 6) Adding a Description
-
-Underneath on a new line, we'll add a description by typing `MVP hotline
-answerer`.
-
-> ![](img/html_add_description.gif)
+### 1) Creating the CSS File
 
-Notice that the blank line between the name and description isn't reflected on
-the web page. Adding extra blank lines or spaces between words in HTML does will
-not change what the website looks like.
+We'll now make an `styles.css` file. Right click the sidebar, select "New
+File," and name the file "`styles.css`."
```

Comment:
Actually this should be "styles.css." with no code fence, but because periods go inside quotes i didn't want people to name their file "`styles.css.`"
Don't know how to deal with this so I'll just use code fence.